src/vue/components/panel-compact.vue

```diff
@@ -268,107 +323,80 @@ export default {
     infoIcon: InfoIcon,
     grammarIcon: GrammarIcon,
     treebankIcon: TreebankIcon,
+    wordUsageIcon: WordUsageIcon,
     alphTooltip: Tooltip,
     lookup: Lookup,
-    reskinFontColor: ReskinFontColor
+    reskinFontColor: ReskinFontColor,
+    wordUsageExamplesBlock: WordUsageExamplesBlock
   },
   directives: {
     onClickaway: onClickaway
   },
+  positionClassVariants: {
+    left: 'alpheios-panel-left',
```

Comment:
if this is the base panel that will be used for mobile (which appears as split screen, rather than right/life) then should the left/right and panelPosition, etc. all be moved to the panel.vue?

Reply:
Yes, agree, this shall be moved to `panel.vue`, will do that. Great suggestion, thanks!